assignments/callbacks.js

```diff
@@ -24,29 +24,72 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+   return cb(arr[arr.length -1]);
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  let sum = x + y;
+   return cb(sum);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  let product =  x*y;
+  return cb(product);
```

Comment:
these absolutely work as is, but you can optimize it even further by just putting the added/multiplied numbers directly in the callback function instead of in 2 lines -- `return cb(x+y)` will interpret what the sum of x and y is before it actually gets run in the callback, for instance